prompts/sources.go

```diff
@@ -109,6 +112,9 @@ func getOverlayPrompts(promptForOverlay *bool, overlayLocation, authHeader, auth
 	return groups
 }
 
+//go:embed sample_openapi.yaml
+var sampleSpec string
```

Comment:
I like the embedding idea. We just need to make sure we keep an eye on keeping the source up to date. 

Not worried about it though